src/main/edu/colostate/cs/cs414/ByteMe/banqi/server/UserNode.java

```diff
@@ -179,6 +179,7 @@ public void OnEvent(Event e, TCPConnection connect) throws IOException {
 			String invFrom = new String(inF);
 			gamesInvitedTo.add(invFrom);
 			System.out.println("\nYou received an invite from " + invFrom + "!");
+			System.out.println("To respond to this invite, enter 2.\n");
```

Comment:
This only works if the user is on the main menu to press 2. They can be anywhere else and get this notification and '2' will not work for the other menu they are on.

Reply:
Well, then what do they use in that case?

Reply:
Is it exit and then 2?

Reply:
They go to the manage invites screen to accept the invite

Reply:
I know that.  You enter 2 to get to the manage invites screen.  If you are not on the main menu, you enter 'exit' then it takes you back to the main menu and then you can enter 2 to manage invites.

Reply:
How about this?

	System.out.println("To respond to this invite, enter '2'.\n");
	System.out.println("If you are not on the main menu, enter 'exit' and then '2'.\n");

Reply:
It can be exit and then 2. Unless they are in the menu to view user profiles.

Reply:
Exit and 2 works the same for me on that menu too.  Is this the menu you're talking about?
```

scoobs's Profile

Joined: 2018/10/27 17:45:45
Wins: 0
Losses: 0
Draws: 0
Forfeits: 0

1) Search for player
To exit, type 'exit' and press Enter
```

Reply:
One more deeper. Press one on that screen.

Reply:
Well, these statements are not a perfect solution, but most of the time a user will not be mid-search for a second profile when they receive a notification.  So I feel like this covers most of the cases.

Reply:
I've updated it to have the 2 and the exit, 2 options.